gpu/README.md

```diff
@@ -65,6 +65,11 @@ gpu-driver-provider=<OS|NVIDIA>` metadata value.
         --metadata install-gpu-agent=true \
         --scopes https://www.googleapis.com/auth/monitoring.write
     ```
+#### GPU Scheduling in Yarn:
+
+Yarn is the default resouce manager for DataProc. To use GPU scheduling feature in Spark, it requires YARN version >= 2.10 or >=3.1.1. If intended to use spark with Deep Learning use case, it recommended to use Yarn >= 3.1.3 to get support for nvidia-docker version 2. 
```

Comment:
Please, link https://github.com/NVIDIA/nvidia-docker repo to `nvidia-docker` mention

Reply:
fixed